Sources/StreamVideo/WebRTC/WebRTCClient.swift

```diff
@@ -1117,6 +1122,19 @@ class WebRTCClient: NSObject, @unchecked Sendable {
         )
     }
     
+    private func subscribeToThermalChanges() {
+        thermalStateCancellable?.cancel()
+
+        thermalStateCancellable = ThermalStateObserver
+            .shared
+            .$state
+            .removeDuplicates()
+            .sink { [weak self] _ in
+                Task { [weak self] in
+                    try await self?.updateParticipantsSubscriptions()
```

Comment:
why do we need to update participant subscriptions here? The `renderingFramesPerSecond` is only applied in the rendering views, so calling `updateParticipantsSubscriptions` won't have any effect.